featuretools/tests/conftest.py

```diff
@@ -380,7 +380,7 @@ def label_func(df):
         return df['value'].sum() > 10
 
     lm = cp.LabelMaker(
-        target_entity='id',
+        target_dataframe='id',
```

Comment:
This is a compose parameter, right? Maybe this doesn't need updating yet?

Reply:
Yeah this can wait. I filed an issue to track